src/test/java/org/jenkinsci/plugins/ghprb/GhprbTestUtil.java

```diff
@@ -372,11 +362,12 @@ private static void setupReq() throws Exception{
         given(req.bindJSON(any(Class.class), any(Class.class), any(JSONObject.class))).willCallRealMethod();
         given(req.setBindInterceptpr(any(BindInterceptor.class))).willCallRealMethod();
```

Comment:
setBindIntercept_**o**_r

I spent hours battling this NPE, I knew it was a mocking problem but couldn't see it. Now looking at your PR the typo in setBindinterceptpr is blindingly obvious. I checked out your PR locally and all the tests pass 🎉   :shipit: